model/shared/src/main/scala/explore/model/allocations.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.Order.given
+import cats.derived.*
+import io.circe.Decoder
+import io.circe.generic.semiauto.*
+import lucuma.core.enums.Partner
+import lucuma.core.enums.ScienceBand
+import lucuma.core.util.NewType
+import lucuma.core.util.TimeSpan
+import lucuma.odb.json.time.decoder.given
+import monocle.Focus
+import monocle.Lens
+
+import scala.collection.immutable.SortedMap
+
+case class Allocation(partner: Partner, scienceBand: ScienceBand, duration: TimeSpan) derives Eq
```

Comment:
Should this be in core?

Reply:
I'm not sure. Will it be used from somewhere else? We also have some model ADTs in schemas.